modules/hal_hpmicro/Kconfig

```diff
@@ -0,0 +1,86 @@
+# Copyright 2022 hpmicro
+# SPDX-License-Identifier: Apache-2.0
+config ZEPHYR_HAL_HPMICRO_MODULE
+	bool
+
+config HAS_HPMICRO_HAL
+	bool "HPMSDK HAL drivers support"
+
+if HAS_HPMICRO_HAL
+
+config HAS_HPMSDK_UART
+	bool "HPMSDK UART driver support"
+config HAS_HPMSDK_DRAM
+	bool "HPMSDK DRAM driver support"
+config HAS_HPMSDK_SDP
```

Comment:
again, `HAS_` doesn't look right. These options are being used to device which HAL drivers are compiled here https://github.com/hpmicro/zephyr_hal_hpmicro/blob/manifest-rev/hpm_sdk/drivers/CMakeLists.txt, please, to be consistent with other platforms use `USE_`. Drivers should be responsible to select what they need. For example, the UART driver will `select USE_HPMSDK_UART`.

Reply:
OK, will make changes.